src/main/java/uk/gov/justice/digital/hmpps/keyworker/integration/RetryWebClientException.kt

```diff
@@ -16,3 +17,14 @@ fun <T> Mono<T>.retryRequestOnTransientException(): Mono<T> =
         signal.failure()
       },
   )
+
+fun <T> Flux<T>.retryRequestOnTransientException(): Flux<T> =
```

Comment:
ManageUsersClient.getUsersDetails has a similar pattern but doesn't look like it has retry on it. Could you check if they can both use the same pattern